unrpyc.py

```diff
@@ -262,43 +262,34 @@ def main():
             print("\n".join(e.args))
             return
 
-    # Expand wildcards
-    def glob_or_complain(s):
-        retval = glob.glob(s)
-        if not retval:
-            print("File not found: " + s)
-        return retval
-    filesAndDirs = [glob_or_complain(i) for i in args.file]
-    # Concatenate lists
-    filesAndDirs = list(itertools.chain(*filesAndDirs))
-
-    # Recursively add .rpyc files from any directories passed
-    files = []
-    for i in filesAndDirs:
-        if path.isdir(i):
-            for dirpath, dirnames, filenames in walk(i):
-                files.extend(path.join(dirpath, j) for j in filenames if len(j) >= 5 and j.endswith(('.rpyc', '.rpymc')))
-        else:
-            files.append(i)
-
-    # Check if we actually have files. Don't worry about
-    # no parameters passed, since ArgumentParser catches that
-    if len(files) == 0:
-        print("No script files to decompile.")
+    def rpyc_check(inp):
+        return bool(inp.suffix in ('.rpyc', '.rpymc') and inp.is_file())
+
+    files = list()
+    for item in args.file:
+        if item.is_dir():
+            for entry in item.rglob('*'):
+                if rpyc_check(entry):
+                    files.append(entry)
+        elif rpyc_check(item):
+            files.append(item)
+
+    # Check if we actually have files. Don't worry about no parameters passed,
+    # since ArgumentParser catches that
+    if not files:
+        print("Found no script files to decompile.")
         return
 
-    files = [(args, x, path.getsize(x)) for x in files]
-    processes = int(args.processes)
-    if processes > 1:
-        # If a big file starts near the end, there could be a long time with
-        # only one thread running, which is inefficient. Avoid this by starting
-        # big files first.
-        files.sort(key=itemgetter(2), reverse=True)
-        results = Pool(int(args.processes), sharelock, [printlock]).map(worker, files, 1)
+    files.sort(key=lambda x: x.stat().st_size, reverse=True)
+
+    # changes: contextmanager; passing filesize over is unneeded; only use multiprocessing if
```

Comment:
Remove the `changes:` part. This is a commit message, not a comment. If it becomes hard to figure out what the commit message is pointing to, you should probably do smaller commits.